functions/go/set-namespace/README.md

```diff
@@ -0,0 +1,20 @@
+# One time setup
+```
+go mod init
+go get sigs.k8s.io/kustomize/kyaml
+```
+
+# Build the binary
+```
+go build .
+```
+
+# Test the binary
+```
+kpt fn source config/ | kpt fn run --enable-exec --exec-path ./set-namespace -- namespace=test-ns
```

Comment:
Can you add [e2e tests](https://github.com/GoogleContainerTools/kpt-functions-catalog/blob/master/tests/e2e.sh#L106)? This can be one of them.

Reply:
A e2e test seems to require a published docker image for `kpt fn run`. 
should we add e2e tests for this after the image release workflow is done?

Reply:
I'd rather have e2e tests using a docker image now since that's how our users will access them. We also run e2e script regularly which will catch regressions faster than manually testing the binary. So let's have an e2e test for this success condition and perhaps one error condition. I manually created a docker image at gcr.io/kpt-functions/set-namespace which you can use. I also filed a bug to create a release workflow https://github.com/GoogleContainerTools/kpt/issues/781.